examples/example-dsl/src/main/scala/com/lightbend/kafka/scala/iq/example/WeblogProcessing.scala

```diff
@@ -167,24 +170,27 @@ object WeblogProcessing extends WeblogWorkflow with AppSerializers with FailFast
     filtered(0).mapValues {
       case ValidLogRecord(r) => r
       case _ => ??? // should never happen since we pre-emptively filtered with `branch`
-    }.to(config.toTopic, Produced.`with`(byteArraySerde, logRecordSerde))
+    }.to(config.toTopic)  // will use default serde
```

Comment:
This will use default serde from config since the implicit for `LogRecordSerde` is not in scope.